docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md

```diff
@@ -128,7 +128,7 @@ To communicate with your LKE cluster, kubectl requires a copy of your cluster's
 1. Access your LKE cluster's kubeconfig file by sending a `GET` request to the `/lke/clusters/{clusterId}/kubeconfig` endpoint. Ensure you replace `12345` with your cluster's ID that you recorded in the previous section:
 
         curl -H "Authorization: Bearer $TOKEN" \
-          https://api.linode.com/v4beta/lke/clusters/12345/kubeconfig
+          https://api.linode.com/lke/clusters/12345/kubeconfig
```

Comment:
Shouldn't the url be api.linode.com/**v4**/lke?

Reply:
fixed!